custom_components/tuya_local/devices/klarta_stor2_purifier.yaml

```diff
@@ -0,0 +1,149 @@
+name: Klarta Stor 2 air purifier
+products:
+  - id: ku6ae3y8ad6tbx3b
+    name: "KLARTA STOR 2 "
+primary_entity:
+  entity: switch
+  icon: "mdi:air-purifier"
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+secondary_entities:
+  - entity: sensor
+    class: pm25
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        class: measurement
+        unit: ugm3
+  - entity: select
+    name: Mode
+    dps:
+      - id: 3
+        name: option
+        type: string
+        mapping:
+          - dps_val: auto
+            value: "Auto"
+          - dps_val: "manual"
+            value: "Manual"
+          - dps_val: "sleep"
+            value: "Sleep"
+          - dps_val: "screenoff"
+            value: "Turn off LEDs"
+  - entity: select
+    name: Speed
+    dps:
+      - id: 4
+        name: option
+        type: string
+        icon: "mdi:fan"
+        mapping:
+          - dps_val: speedmode1
+            value: "Silent"
```

Comment:
When converted to a fan entity, these should be mapped to percentages. (17, 33, 50, 67, 83, 100)